opi/__init__.py

```diff
@@ -440,12 +438,12 @@ def ask_for_option(options, question="Pick a number (0 to quit):", option_filter
 		If needed, a pager will be used, unless disable_pager is True.
 	"""
 
-	padding_len = math.floor(math.log10(len(options)))
+	padding_len = len(str(len(options)))
 	i = 1
 	numbered_options = []
-	terminal_width = os.get_terminal_size().columns-1 if sys.stdout.isatty() else 0
+	terminal_width = os.get_terminal_size().columns - 1 if sys.stdout.isatty() else 0
 	for option in options:
-		number = "%%%id. " % (padding_len+1) % i
+		number = f'{i:{padding_len}}. '
```

Comment:
```suggestion
		number = f'{i:{padding_len + 1}}. '
```

Reply:
Changed in L488 instead, as it is always applied.